src/style.css

```diff
@@ -0,0 +1,110 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+form {
+  margin-top: 20px;
+}
+
+.search {
+  box-sizing: border-box;
+  width: 100%;
+  font-family: Avenir, sans-serif;
+  color: #3d4e61;
+}
+
+.search--big {
+  height: 70px;
+  font-size: 16px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
```

Comment:
Use css variables for colors. Check other cases.


[More info](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)